test/bitonic_sort_test.dart

```diff
@@ -0,0 +1,483 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// bitonic_sort_test.dart
+// Tests for bitonic sort
+//
+// 2023 May 3
+// Author: Yao Jing Quek <yao.jing.quek@intel.com>
+//
+
+// ignore_for_file: cascade_invocations
+
+import 'dart:async';
+import 'dart:math';
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:test/test.dart';
+
+Future<void> main() async {
+  tearDown(() async {
+    await Simulator.reset();
+  });
+
+  List<int> generateNonDuplicateRand(int seed, int maxLength, int listLength) {
```

Comment:
I think can go with `int` first since the `List` provides `.sort()` function which sorts the generated numbers. I am not sure do we have sort function that can sort `LogicValue` to ascending order.

Reply:
```dart
  type '_SmallLogicValue' is not a subtype of type 'Comparable<dynamic>' in type cast
  dart:collection                     ListBase.sort
  test/bitonic_sort_test.dart 193:19  main.<fn>.<fn>.<fn>
```

Is this an issue in ROHD? I get this error when trying to use `.sort()` on List of `LogicValue`. Probably want to disable calling `.sort()` or implement this function.

Reply:
I'm not sure if it's a bug, but that's definitely a missing feature.  We should implement it though!  Filed here: https://github.com/intel/rohd/issues/371

Reply:
I will mark this resolved first temporary since the test is not critical.